CONTRIBUTING.md

```diff
@@ -59,7 +59,7 @@ Also, the WASM example compilation should be checked:
 
 ```bash
 rustup target add wasm32-unknown-unknown
-cargo check --example web_app --target wasm32-unknown-unknown --features=sync
+cargo check -p web_app --target wasm32-unknown-unknown
```

Comment:
This changes are not impactful here?

Reply:
No, they needed to be changed back when tamo changed the wasm process but it was forgotten.